CONTRIBUTOR-DOCS/04_workstream-info/02_2nd-gen-component-migration/02_step-by-step/06_add-stories-for-2nd-gen-component.md

```diff
@@ -0,0 +1,24 @@
+<!-- Generated breadcrumbs - DO NOT EDIT -->
+
+[CONTRIBUTOR-DOCS](../../../README.md) / [Workstream Info](../../README.md) / [2nd-gen Component Migration](../README.md) / Step By Step / Add stories for 2nd-gen component
+
+<!-- Document title (editable) -->
+
+# Add stories for 2nd-gen component
+
+<!-- Document content (editable) -->
+
+- Create `stories/[component].stories.ts` file
+- Add section headers: METADATA, STORIES, HELPER FUNCTIONS (if needed)
+- In METADATA section:
+    - Import `StoryObj as Story` for consistency
+    - Set up Storybook helpers with `getStorybookHelpers()`
+    - Configure argTypes for component-specific controls
+    - Define and export the meta object
+- In STORIES section:
+    - Create Default story (without `tags: ['!dev']`)
+    - Create additional stories showcasing variants, sizes, states
+    - Add `tags: ['!dev']` to all non-default stories
+- In HELPER FUNCTIONS section (if needed):
+    - Add any utility functions or template helpers
```

Comment:
I think I see that some of the utility functions here are a little repetitive... for instance, Status light and Badge both use a `CONTAINER` function. Status light implements a `sizeMap` function that maybe could be used in some of the other components in barebones, but definitely could be used in other components outside of barebones as we go along.

I believe that the repetitiveness rather than creating utility functions in a place where they'd be reusable is a deliberate choice for now, right? Could be worth mentioning that here or somewhere if we continue with this for more component migrations--to me, one challenge with this work was trying to figure out what's in scope and what's not.